tests/contrib/unittest_plugin/test_unittest.py

```diff
@@ -0,0 +1,448 @@
+import sys
+import unittest
+
+import pytest
+
+from ddtrace.constants import ERROR_MSG
+from ddtrace.constants import ERROR_TYPE
+from ddtrace.constants import SPAN_KIND
+from ddtrace.contrib.unittest.constants import COMPONENT_VALUE
+from ddtrace.contrib.unittest.constants import FRAMEWORK
+from ddtrace.contrib.unittest.constants import KIND
+from ddtrace.contrib.unittest.patch import _set_tracer
+from ddtrace.contrib.unittest.patch import patch
+from ddtrace.ext import SpanTypes
+from ddtrace.ext import test
+from ddtrace.internal.constants import COMPONENT
+from tests.utils import TracerTestCase
+from tests.utils import override_env
+
+
+class UnittestTestCase(TracerTestCase):
```

Comment:
These tests look good. Can we add one or two snapshot tests to validate the structure of the tracer generated by the unittest integration: https://ddtrace.readthedocs.io/en/stable/contributing-integrations.html#snapshot-tests. Let me know if you need help with this, the doc linked above is new